### PR TITLE
Undo rollbacks when unbanned.

### PR DESCRIPTION
**Pull request by OxLemonxO**
Solve #835

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java [hunk 1]
```diff
@@ -98,6 +98,9 @@ else if (args.length == 2)
             }
             else if (mode.equalsIgnoreCase("unban"))
             {
+                if(TFM_RollbackManager.canUndoRollback(username) {
+                TFM_RollbackManager.undoRollback(username);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java]: Incorrect indentation here.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java [hunk 2]
```diff
@@ -98,6 +98,9 @@ else if (args.length == 2)
             }
             else if (mode.equalsIgnoreCase("unban"))
             {
+                if(TFM_RollbackManager.canUndoRollback(username) {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java]: Formatting.

**Comment by OxLemonxO** [comment]
Fixed, @JeromSar

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java [hunk 3]
```diff
@@ -98,6 +98,10 @@ else if (args.length == 2)
             }
             else if (mode.equalsIgnoreCase("unban"))
             {
+                if (TFM_RollbackManager.canUndoRollback(username) 
+                {
+                TFM_RollbackManager.undoRollback(username);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_glist.java]: Incorrect indentation.

**Comment by OxLemonxO** [comment]
fixed now? @JeromSar

**Comment by JeromSar** [comment]
@OxLemonxO This PR will not compile.

**Comment by OxLemonxO** [comment]
Why won't it?

**Comment by OxLemonxO** [comment]
Fixed

**Comment by OxLemonxO** [comment]
@JeromSar This doesn't need CLA anymore.

**Comment by DUFCLiam** [comment]
@OxLemonxO You may want to add a calendar bit there so if they are not un-banned within 1 hour it is aborted.

**Comment by JeromSar** [comment]
This pull request does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and therefore is closed.

**Pull request closed without merging**